content/docs/0.8.x/reference/cli/_index.md

```diff
@@ -166,4 +166,29 @@ keptn.exe auth --endpoint=%KEPTN_ENDPOINT% --api-token=%KEPTN_API_TOKEN%
 </p>
 </details>
 </p>
-</details>
\ No newline at end of file
+</details>
+
+## Use Keptn CLI with Multiple Contexts
+
+After authenticating keptn CLI with multiple kubernetes clusters, we can directly run keptn CLI commands in the current context. As soon as we switch the kube context (e.g. `kubectl config use-context my-cluster-name`), Keptn CLI will detect the change in the context and ask you to run the command in the changed context or not. If the Keptn CLI is already authenticated for that particular context, the command will be ran successfully else it will end up throwing error i.e `Error: credentials not found in native keychain`
+
+In case of multi-installation of keptn in the same cluster, we need to pass a flag `--namespace` or `-n` to tell the keptn CLI to use the credentials for that particular keptn installation, else it will take the default namespace i.e `keptn`
```

Comment:
Minor changes - please replace the paragraph with:

```
In case of multi-installation of Keptn in the same cluster, we need to pass a flag `--namespace` or `-n` to tell the Keptn CLI to use the credentials for that particular Keptn installation, else it will take the default namespace which is: `keptn`
```